node/core/bitfield-signing/src/lib.rs

```diff
@@ -45,24 +45,24 @@ const LOG_TARGET: &str = "bitfield_signing";
 pub struct BitfieldSigningJob;
 
 /// Errors we may encounter in the course of executing the `BitfieldSigningSubsystem`.
-#[derive(Debug, Error)]
+#[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
 pub enum Error {
-	/// error propagated from the utility subsystem
```

Comment:
Why remove the variant documentation on this enum?

Reply:
If the error variant needs documentation, then the error message needs to be improved. There should be nothing required to understand the error message.

Reply:
This is inline with changes in substrate and polkadot landed earlies.